WBooks/Repositories/User+BooksRepository.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  User+BooksRepository.swift
+//  WBooks
+//
+//  Created by Carolina Arcos on 9/4/18.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import Foundation
+import ReactiveSwift
+import Networking
+
+protocol UserBooksRepositoryType {
+    
+    var userRepository: UserRepositoryType { get }
+    var booksRepository: BooksRepositoryType { get }
+}
+
+class UserBooksRepository: AbstractRepository, UserBooksRepositoryType {
+    
+    var userRepository: UserRepositoryType = NetworkingBootstrapper.shared.createUserRepository()
+    var booksRepository: BooksRepositoryType = NetworkingBootstrapper.shared.createBooksRepository()
```

Comment:
This is not perfectly okay. If this `UserBooksRepository` is created with a networkConfiguration and a sessionManager, its child repositories should be created with that variables as well, if not it wouldn't be necessary to receive those variables in the init